must-gather/collection-scripts/pre-install.sh

```diff
@@ -36,10 +39,12 @@ deploy(){
           fi
      fi
 
-     for node in ${nodes}; do
+    if [ "$storageClusterPresent" ]; then
+    for node in ${nodes}; do
```

Comment:
the creation of debug pod is independent of storagecluster.

Reply:
fixed